examples/official-storybook/stories/core/layout.stories.js

```diff
@@ -1,43 +1,35 @@
 import React from 'react';
-import { Button } from '@storybook/components';
+
+// eslint-disable-next-line react/prop-types
+const Box = ({ children, display = 'block' }) => (
+  <div style={{ display, border: '2px solid #FF4785', padding: 10 }}>{children}</div>
+);
 
 export default {
   title: 'Core/Layout',
   parameters: {
-    layout: 'padded',
+    layout: ctx => ctx.name.toLowerCase().split(' ')[1],
```

Comment:
I'd rather not do this, until it's really required/requested from the community.

I think this is too "magical" tbh.

Reply:
Yeah okay, maybe not in official-storybook. I did this to demonstrate how the callback works. I guess a (unit) test is a better place for it.

Reply:
I just don't foresee people needing the callback, and it's another bit of complexity.

Functions are harder to share between manager & preview. So I'd rather not add this function option if it's not required by the community.